General cleanup

karimcambridge opened the pull request:
Fix CSS.
Remove unnecessary font.
Change example from saarp to example.org or w/e.
Spaces -> tabs.

karimcambridge committed:
General cleanup

Fix CSS.
Remove unnecessary font.
Change example from saarp to example.org or w/e.
Spaces -> tabs.

rt-2 commented on html/dist/voiceChat.css:
```diff
@@ -1,17 +1,12 @@
-@font-face {
-    font-family: "GTAV_font_6";
-    src: url('./font6.ttf') format("truetype");
-}
```
You told me on discord that the font was not there, but actually it IS int he dist folder

karimcambridge replied:
Fixed.

rt-2 commented on html/dist/voiceChat.css:
```diff
@@ -25,7 +20,6 @@
     position: relative;
     width: 100%;
     height: auto;
-    font-family: GTAV_font_6;
```
We should try to fix this

karimcambridge replied:
Fixed.

karimcambridge committed:
Consistent coding style on client

guiReady -> playerReady, guiReady will be removed in 0.4.
Send a test message when browser creation failed.
Spaces -> tabs.
Add default `MAX_PLAYERS` variable.
Toggle -> toggleVoiceChat.
_PosDistanceFromPos ->  mp.game.gameplay.getDistanceBetweenCoords.
Sanity checks on entity streaming!

rt-2 commented on html/dist/voiceChat.css:
```diff
@@ -1,17 +1,12 @@
 
 #iframes_container {
-	//display: none;
+	display: none;
```
https://github.com/rt-2/RageMP-SAARPV-voiceChat/blob/master/html/index.html#L107 is repeating it btw

karimcambridge replied:
Fixed.

karimcambridge committed:
Fix exports issue

rt-2 reviewed (commented):
Nice job overall! We need to figure out what,s with the font tho...
Thank you
rt-2

karimcambridge committed:
Remove inline html css

karimcambridge committed:
Readded font

rt-2 reviewed (approved):
Very nice job, thank you for your time Kar!
